pkg/common/log/example_log_test.go

```diff
@@ -139,3 +144,50 @@ func processStep3InLOG(id string) {
 func getServiceLogger(serviceType metadata.ServiceType, uuid string) *log.MOLogger {
 	return log.GetServiceLogger(logutil.GetGlobalLogger(), serviceType, uuid)
 }
+
+func ExampleMOLogger_WithContext() {
+	ctx, span := trace.Start(context.Background(), "ExampleMOLogger_WithContext")
+	defer span.End()
+	logger := getServiceLogger(metadata.ServiceType_CN, "cn0").WithContext(ctx)
+	logger.Info("info log 1, with Context in MOLogger.")
+	logger.Info("info log 2, with Context in MOLogger.")
+	// 2022/12/17 16:37:22.995805 +0800 INFO cn-service log/example_log_test.go:152 info log 1, with Context in MOLogger. {"uuid": "cn0", "span": {"trace_id": "349315cb-2044-f2dd-b4b1-e20365c8a944", "span_id": "fd898a22b375f34e"}}
+	// 2022/12/17 16:37:22.995812 +0800 INFO cn-service log/example_log_test.go:153 info log 2, with Context in MOLogger. {"uuid": "cn0", "span": {"trace_id": "349315cb-2044-f2dd-b4b1-e20365c8a944", "span_id": "fd898a22b375f34e"}}
+}
+
+func ExampleLogOptions_WithContext() {
+	ctx, span := trace.Start(context.Background(), "ExampleLogOptions_WithContext")
+	defer span.End()
+	logger := getServiceLogger(metadata.ServiceType_CN, "cn0")
+	logger.Log("this is an info log 1, with Context in LogOptions.", log.DefaultLogOptions().WithLevel(zap.InfoLevel).WithContext(ctx))
+	logger.Log("this is an info log 2, without Context.", log.DefaultLogOptions().WithLevel(zap.InfoLevel))
+	// 2022/12/17 16:37:22.995817 +0800 INFO cn-service log/example_log_test.go:162 this is an info log 1, with Context in LogOptions. {"uuid": "cn0", "span": {"trace_id": "9f0907c7-7fa6-bb7c-2a25-384f52e03cd5", "span_id": "068f75a50921c85f"}}
+	// 2022/12/17 16:37:22.995820 +0800 INFO cn-service log/example_log_test.go:163 this is an info log 2, without Context. {"uuid": "cn0"}
+}
+
+func TestExample(t *testing.T) {
```

Comment:
move this Unit Test to a single test file. The example_log_test is used to go docs

Reply:
removed